main.rb

```diff
@@ -0,0 +1,39 @@
+def word_decoder(word)
+  alphabet = { '.-': 'A', '-...': 'B', '-.-.': 'C', '-..': 'D',
+               '.': 'E', '..-.': 'F', '--.': 'G', '....': 'H',
+               '..': 'I', '.---': 'J', '-.-': 'K', '.-..': 'L',
+               '--': 'M', '-.': 'N', '---': 'O', '.--.': 'P',
+               '--.-': 'Q', '.-.': 'R', '...': 'S', '-': 'T',
+               '..-': 'U', '...-': 'V', '.--': 'W', '-..-': 'X',
+               '-.--': 'Y', '--..': 'Z' }
+
+  result = word
+  result = alphabet[word.to_sym] if word != '/'
+  result
+end
+
+def message_decoder(message)
+  decoded_words = []
+  words_arr = message.split
+  words_arr.each do |word|
+    decoded_word = word_decoder(word)
+    decoded_words.push(decoded_word)
+  end
+  decoded_words.join
+end
+
+def morse_decoder(message)
+  decoded_messages = []
+  messages_arr = message.split('   ')
+  messages_arr.each do |msg|
+    decoded_message = message_decoder(msg)
+    decoded_messages.push(decoded_message)
+    decoded_messages.push(' ')
+  end
+  decoded_messages.join
+end
+
+morse = '      .-   -... --- -..-   ..-. ..- .-.. .-..   --- ..-. / .-. ..- -... .. . ...
+'
+translate = morse_decoder(morse)
+puts translate
```

Comment:
- [ ] Let's add a proper naming as requested in the [guidelines](https://github.com/Metsanpeitto/ruby_morse_m4_w2/tree/feature-1) for your project. 

<img width="861" alt="1" src="https://user-images.githubusercontent.com/58504655/135318147-4cb27c96-29bc-457c-9b0f-154562de9888.PNG">


- [ ] Let's add these lines of code in main.rb to run it properly. 

```
p decode_char('.-')
p decode_word('-- -.--')
p decode("-- -.--   -. .- -- .")
p decode(".-   -... --- -..-   ..-. ..- .-.. .-..    --- ..-.    .-. ..- -... .. . ...")
```

Reply:
:+1:  Ok, Done